impl/src/main/java/org/jboss/cdi/tck/tests/full/context/dependent/DependentContextTest.java

```diff
@@ -46,4 +47,14 @@ public void testDependentsDestroyedWhenElEvaluationCompletes() throws Exception
         assert FoxRun.isDestroyed();
         assert Fox.isDestroyed();
     }
+
+    @Test
+    @SpecAssertion(section = DEPENDENT_CONTEXT, id = "g")
+    // Dependent context is now always active
+    public void testContextIsActiveWhenEvaluatingElExpression() {
```

Comment:
TBF I am not sure this test is even worth having since I cannot imagine an implementation now having dependent context always active... but then again, what do I know? :-)